scripts/ci/build-contract.sh

```diff
@@ -9,7 +9,7 @@ set -eux
 set -o pipefail
 
 CONTRACT=$(basename $1)
-SIZE_OUT=$(RUST_LOG="" cargo +nightly contract build --release --manifest-path $1/Cargo.toml --output-json) || exit $?
+SIZE_OUT=$(RUST_LOG="" cargo contract build --release --manifest-path $1/Cargo.toml --output-json) || exit $?
```

Comment:
```suggestion
SIZE_OUT=$(RUST_LOG="" cargo +stable contract build --release --manifest-path $1/Cargo.toml --output-json) || exit $?
```

The CI images still use `nightly` as the default toolchain. This is still gonna break though due to the `--output-json` + `dylint` bug that I mentioned elsewhere

Reply:
Stable as default is being added in paritytech/scripts#458.